template/emails/catalog_sync_status.html

```
@@ -0,0 +1,278 @@
+<!-- REQUIRED PARAMS: Bot ID, Execution ID, Sync Status, Message -->
+<!-- Sub: Catalog Sync Status Update -->
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title></title>
```

HTMLHint (1.5.0)

[error] <title></title> must not be empty.

(title-require)